corehq/apps/users/models_role.py

```diff
@@ -111,6 +115,11 @@ def create(cls, domain, name, permissions=None, assignable_by=None, **kwargs):
 
         return role
 
+    @classmethod
+    @quickcache(['domain'])
+    def commcare_user_default(cls, domain):
+        return UserRole.objects.get_commcare_user_default(domain)
```

Comment:
having it on the manager is nice because that is where the actual query is defined and it can be reused elsewhere. I added the method on the class because I wanted the value to be cached which I couldn't (shouldn't) do at the manager level.